x-pack/test/security_solution_endpoint_api_int/services/roles_users.ts

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Maybe consider moving this service to `x-pack/test/security_solution_ftr/services` since it will be used in multiple test suites and not only `security_solution_api_int`?

Reply:
👍